### ISSUE TITLE
Not able to see tracebacks

### ISSUE THREAD

**Issue by dvl** [django/channels#846]
Hi,

I'm trying to upgrade my project to channels 2 but I've noticed a weird behavior when an exception is raised on a consumer I'm not able to see the traceback in nowhere.

```python
class BidConsumer(JsonWebsocketConsumer):

    def receive_json(self, content):
        print(content['non_existing_key'])  # KeyError is expected
```

I just noticed the error because I lost the connection to websocket on browser but I can't see anything on console.

I'm using Python 3.6.4, Django 2.0.2, Channels 2.0.0 and since I'm still testing I didn't configure anything as ASGI backend.

This only happens when Channels are running via runserver if I use via daphne I'm able to see exceptions as expected.

**Comment by andrewgodwin** [comment]
Hmm, maybe the logging is absorbing it. I'll look into it later.

**Comment by andrewgodwin** [comment]
I can't replicate this locally - the tracebacks all seem to come through fine, on both `runserver` and plain `daphne`.

Could you confirm the following for me?

* You don't have any logging overrides that might stop the error messages being shown? They are sent to the logger as a message with severity ERROR, _not_ printed to stderr directly.

* This behaviour is also the case in a basic SyncConsumer and AsyncConsumer?

* Does it suppress errors in `connect()` too?

**Comment by dvl** [comment]
Maybe I have found something strange on daphne side...

https://github.com/django/daphne/blob/master/daphne/server.py#L192-L200

You're using logging (the module) instead of logger (the instance) when an error occur, this is intended? If I use logger.error instead of logging.error It's works.

**Comment by andrewgodwin** [comment]
That is a bug, but shouldn't prevent things from being printed unless there's a global log override somewhere. I'll fix that now.

**Comment by andrewgodwin** [comment]
Try running Daphne with the above commit and see if it works?

**Comment by dvl** [comment]
It's works! Using the code from master solved the issue.

Thank you

**Comment by andrewgodwin** [comment]
Alright! I've released that fix in `daphne 2.0.1`.
